paper-ripple.html

```diff
@@ -88,14 +88,14 @@
       transform: translate3d(0, 0, 0);
     }
 
-    paper-ripple[noink] {
+    :host([noink]) {
       pointer-events: none;
     }
 
-    paper-ripple #background,
-    paper-ripple #waves,
-    paper-ripple .wave-container,
-    paper-ripple .wave {
+    :host #background,
```

Comment:
Nuke all `:host` prefixes. They aren't needed in shadow DOM, and they're polyfilled in shady.